src/_cffi_src/openssl/src/osrandom_engine.c

```diff
@@ -117,10 +85,10 @@ static int osrandom_rand_bytes(unsigned char *buffer, int size) {
     int res;
     while (size > 0) {
         /* OpenBSD restricts maximum buffer size to 256. */
-        len = size > 256 : 256: size;
+        len = size > 256 ? 256: size;
```

Comment:
oh my, an embarrassing typo. In my defense OpenBSD / getentropy was on the todo list.